src/test/java/api/items/ItemApiExamples.java

```diff
@@ -1589,6 +1585,36 @@ public void statusDatePropertyPresentOnStatusUpdated()
     );
   }
 
+  @Test
+  public void cannotMarkClaimedReturnedItemAsMissing()
+    throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource createdItem = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .withStatus("Claimed returned")
+      .withBarcode("645398607547")
+      .canCirculate());
+
+    String itemStatus = createdItem.getJson().getJsonObject("status")
+      .getString("name");
+    assertThat(itemStatus, is("Claimed returned"));
+
+    JsonObject updatedItem = createdItem.getJson().copy()
+      .put("status", new JsonObject().put("name", "Missing"));
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse,
+      hasValidationError("Claimed returned item can not be marked as missing",
```

Comment:
Should this say `cannot` instead of `can not`?

```suggestion
      hasValidationError("Claimed returned item cannot be marked as missing",
```